examples/ads.amp.html

```diff
@@ -454,6 +455,16 @@ <h2>AdThrive 320x50 banner</h2>
       data-sizes="320x50">
   </amp-ad>
 
+  <h2>AdUnity (300x250 demo banner)</h2>
+  <amp-ad 
+    type="adunity"
+    width="300" height="250"
+    data-au-account="5057446857420" data-au-site="test.site" data-au-section="" data-au-zone="rectangle"
```

Comment:
can we move to separate lines? Also `data-au-section` without the `=""` should be idiomatic way?

Reply:
data-au-section it's a required parameter but can be empty, should i remove it and change the code to handle missing data-au-section?

Reply:
seems to work ok idiomatic way